universal-application-tool-0.0.1/app/models/Account.java

```diff
@@ -14,9 +14,25 @@
 import javax.persistence.Table;
 import services.program.ProgramDefinition;
 
+/**
+ * An EBean mapped class that represents a single user account in CiviForm. Users of all roles have
+ * an {@code Account}.
+ *
+ * <p>When a user logs in for the first time using SSO or fills out an application as a guest,
```

Comment:
FYI: The `Account` is created when they "sign in" as a guest, not when they apply for something.

Reply:
Done